db/migrations/20230629144437_add_table_graffiti_stats.sql

```diff
@@ -0,0 +1,23 @@
+-- +goose Up
```

Comment:
Create / Remove always with `CONCURRENTLY`

Reply:
oriented here: https://github.com/gobitfly/eth2-beaconchain-explorer/blob/master/db/migrations/20230614122901_add_ens_tables.sql

the problem with using concurrently here is that I need to split the file up in transactions and non-transaction-queries (will do)

Reply:
fixed